shopinvader_guest_mode/data/ir_cron.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+    <record forcecreate="True" id="ir_cron_product_pricelist_assortment" model="ir.cron">
+        <field name="name">Shopinvader: archive expired Guest accounts</field>
+        <field eval="False" name="active"/>
```

Comment:
who / what should activate this?

Reply:
ping @Cedric-Pigeon

Reply:
@simahawk IMO this cron should be active by default

Reply:
I always define crons in xml data as inactive by default. It is a habit I always had. It lets the control to the admin to define when it should trigger for the first time and at which frequency.
I am not against putting it active by default.

Reply:
Generally, I sort of agree but then we should remind this in the backend w/ a prominent alert box in the related section/tab. 
In this particular case I don't think is an issue to enable it: no `is_guest` records -> nothing to do -> no possible errors. Am I wrong?